src/redux/innerPopupReducer.js

```diff
@@ -8,7 +8,8 @@ export const POPUP_TYPES = {
   switchPlan: 'switchPlan',
   cancelSwitch: 'cancelSwitch',
   cancelPause: 'cancelPause',
-  resumeSubscription: 'resumeSubscription'
+  resumeSubscription: 'resumeSubscription',
+  editDeliveryDetails: 'editDeliveryDetails'
```

Comment:
do we need it here, as this popup is manage by popupSlice?

Reply:
nope, it's in `popupSlice.ts` so will delete here